crates/cargo-util/src/sha256.rs

```diff
@@ -39,7 +39,7 @@ impl Sha256 {
 
     pub fn finish(&mut self) -> [u8; 32] {
         let mut ret = [0u8; 32];
```

Comment:
I think this whole function can be simplified to avoid the copy:
```rust
self.0.finalize_reset().into()
```

Reply:
Thanks! Done.